### PR TITLE
LINK-1962 | Input field length validation

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Validate maximum length of input fields

## Closes
[LINK-1962](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1962)

[LINK-1962]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1962?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr375-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr375-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr375-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr375-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr375-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr375-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr375-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/signupGroup/__tests__/validation.test.ts [hunk 1]
```diff
@@ -169,6 +186,48 @@ describe('isBelowMaxAge function', () => {
...
+    serviceLanguage: 'fi',
+  };
+
+  test('should return true if contact person is valid', async () => {
+    expect(await testContactPersonSchema(validContactPerson)).toBe(true);
+  });
+
+  const testCases: [Partial<ContactPersonFormFields>][] = [
+    [{ email: '' }],
+    [{ email: 'not-email' }],
```
Comment (harriris-vincit) [inline src/domain/signupGroup/__tests__/validation.test.ts:208]: Here too max length test cases for email and phone number could be added.

Comment (jorilindell) [reply src/domain/signupGroup/__tests__/validation.test.ts:209]: Added test cases for email and phone number max lengths

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr375-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr375-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr375-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Review by harriris-vincit — APPROVED** [review]
LGTM

**Pull request merged**